File: Tete.Models/Relationships/Mentorship.cs
using System;

namespace Tete.Models.Relationships
{

  public class Mentorship
  {

    public Guid MentorshipId { get; set; }

    public Guid LearnerUserId { get; set; }

    public Guid MentorUserId { get; set; }

    public Guid TopicId { get; set; }

    public bool Active { get; set; }

    public DateTime StartDate { get; set; }

    public DateTime EndDate { get; set; }
  }
}

File: Tete.Web/ClientApp/src/app/models/mentorship.ts
export class Mentorship {
  public mentorshipId: string;
  public learnerUserId: string;
  public learnerContact: string;
  public mentorUserId: string;
  public mentorContact: string;
  public topicId: string;
  public active: boolean;
  public createdDate: Date;
  public startDate: Date;
  public endDate: Date;

  public learnerClosed: boolean = false;
  public learnerClosedDate: Date;
  public learnerRating: number;
  public learnerClosingComments: string;

  public mentorClosed: boolean = false;
  public mentorClosedDate: Date;
  public mentorRating: number;
  public mentorClosingComments: string;
}